### PR TITLE
Reduce calls to populate within test suites; Use a small database of towns for CI

### PR DESCRIPTION
**Pull request by joehybird**

### CODE REVIEW

**Review by genglert — CHANGES_REQUESTED** [review]
Add a comment in settings.py about the existence of "geolocation/data/test.towns.france.csv.zip" which can be useful for unit tests

File: creme/settings.py [hunk 1]
```diff
@@ -1474,6 +1474,11 @@
 # It can be an URL or a local file ; zip files are also supported.
 GEOLOCATION_TOWNS = [
     (join(CREME_ROOT, 'geolocation', 'data', 'towns.france.csv.zip'), {'country': 'France'}),
+    # For the CI a lighter version of the file exists with only the "chef-lieu"
```
Comment (genglert) [inline creme/settings.py:1477]: s/CI/unit tests

**Review by genglert — CHANGES_REQUESTED** [review]
Why the option "'--no-populate'" made its come back ?

**Comment by joehybird** [comment]
???

File: creme/settings.py (hunk 1 above)
Comment (joehybird) [reply creme/settings.py]: > Why the option "'--no-populate'" made its come back ?

Merge issue on my local branch -_-

**Review by genglert — CHANGES_REQUESTED** [review]
Can you fix up/squash your commits please (I think 2 commits -- geolocation & test-populate --- are sufficient) ?

**Review by genglert — APPROVED** [review]

**Pull request merged**